.github/workflows/cargo.yml

```diff
@@ -157,7 +170,7 @@ jobs:
       - name: build
         shell: bash
         run: |
-          cargo build --workspace --all-features --all-targets --locked --release
+          cargo build --workspace --all-targets --locked --release
```

Comment:
Is it worth adding a cross-compile step into the workflow as well so it is easier to  know if the arm64 build breaks?

Reply:
Leaving this for a follow-up.